python/osrd_schemas/osrd_schemas/infra.py

```diff
@@ -458,7 +458,7 @@ class RailJsonInfra(BaseModel):
         description="List of operational points of the corresponding infra"
     )
     routes: List[Route] = Field(description="Routes of the infra")
-    switch_types: List[SwitchType] = Field(description="Switch types of the infra")
```

Comment:
Wouldn't "extend**ed** switch types" make more sense? Sorry, that's a fairly large change at this point